src/main/fish/payara/project/Maven.ts

```diff
@@ -78,7 +78,7 @@ export class Maven implements Build {
                             if (artifacts[i].endsWith('.war')
                                 || artifacts[i].endsWith('.jar')
                                 || artifacts[i].endsWith('.rar')
-                                || artifacts[i] === this.getBuildReader().getFinalName()) {
+                                || artifacts[i] === this.getBuildReader().getFinalName().toString()) {
```

Comment:
getFinalName() already returns a string

Reply:
when debugging the application I saw that after calling the getFinalName() the returned type is an array of one possition, that is why when comparing to the artifacts[i] position the result is always false. That is why I used the toString()  method to get the content of the array as a String, then if both Strings are equal this condition should be true

always get false:
![image](https://user-images.githubusercontent.com/279375/203382799-7cd1729b-c909-4ec6-a60e-931d25200ef6.png)

this should work converting the array to String:
![image](https://user-images.githubusercontent.com/279375/203383265-b041ca5e-f327-4d70-985e-f0aaeea40735.png)